### PR TITLE
✨: Add github actions

### PR DESCRIPTION
**Pull request by pelmini**
Resolves Issue #7

### CODE REVIEW

**Review by CMeza99 — CHANGES_REQUESTED** [review]

**Comment by CMeza99** [comment]
Should add `Closes #21` to git commit to automate closing (and linking) of the issue this PR addresses. Do you know how to do multi-line commits. Easiest way it to not do the commit message at the cli, and let git open vim.

see, https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: .github/workflows/tox_tests.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Fizzbuzz Check
+
+on: [push]
```
[the remark this answers is not in the document]
Comment (pelmini) [reply .github/workflows/tox_tests.yml:3]: Would that be the server side of the CI stuff

[the remark this answers is not in the document]
Comment (CMeza99) [reply .github/workflows/tox_tests.yml:3]: yes. this is telling when Github should trigger. just need to append `pull_request` to the list.

**Comment by CMeza99** [comment]
If we want to have a cleaner commit history, I think having one commit and `Closes #21` going in the body of the commit. https://stackoverflow.com/a/42078093/2079786

I retract my comment about using vim for commit msg being easier. using two `-m` is easier. 😛  (see the Stack Overflow answer).

Think you can rebase and squash the commits? Rewriting history! lol

**Comment by CMeza99** [comment]
Wait. The PR is #21. The commit msg should be closing Issue #7. The PR will close when it is merged.

**Comment by pelmini** [comment]
oops my bad. Thanks for catching that!

**Pull request closed without merging**